src/CarbonAware.DataSources/CarbonAware.DataSources.Json/mock/CarbonAware.DataSources.Json.Mocks.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\CarbonAware.DataSources.Mocks\mock\CarbonAware.DataSources.Mocks.csproj" />
```

Comment:
Ties into my comment below about the `DataSource.Registration` project reference. Seems like you'll need it for the implementation of the Json mock too so either would nee to include it here, or as I suggested below, move the reference to the IDataSource mocker `.csproj` so that all the mocks will have the reference trickled down.

Reply:
Removed, per my other comment